src/components/experiment-wrapper/experiment-wrapper.js

```diff
@@ -75,6 +75,24 @@ const ExperimentWrapper = ({ theme }) => {
     }
   }, [selectedRuns, pinnedRun]);
 
+  useEffect(() => {
+    if (!data?.runsList || data.runsList.length === 0) {
+      return;
+    }
+    subscribeToMore({
+      document: NEW_RUN_SUBSCRIPTION,
+      updateQuery: (prev, { subscriptionData }) => {
+        if (!subscriptionData.data || !prev?.runsList) {
+          return prev;
+        }
+        const newRuns = subscriptionData.data.runsAdded;
+        return Object.assign({}, prev, {
+          runsList: [...newRuns, ...prev.runsList],
+        });
+      },
+    });
```

Comment:
I know it's mostly copy/pasted, though some breathing room would be nice here.

```suggestion
    if (!data?.runsList || data.runsList.length === 0) {
      return;
    }
    
    subscribeToMore({
      document: NEW_RUN_SUBSCRIPTION,
      updateQuery: (prev, { subscriptionData }) => {
        if (!subscriptionData.data || !prev?.runsList) {
          return prev;
        }
        
        const newRuns = subscriptionData.data.runsAdded;
        
        return Object.assign({}, prev, {
          runsList: [...newRuns, ...prev.runsList],
        });
      },
    });
```